From 7b31bf4f1b5ee9343b737bad4efefa1c2c2a70e7 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Tue, 20 Feb 2024 14:23:03 +0100
Subject: [PATCH] ARM: configs: add camera related configs in STM32MP1 addons
 fragment

Add OV5640 / GC2145 and ST-MIPID02 as modules

Change-Id: I7c98240553a0fa819a7c0fc0608076cd48a79cbb
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/361981
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Hugues FRUCHET <hugues.fruchet@st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -65,3 +65,6 @@ CONFIG_TOUCHSCREEN_GOODIX=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB_ONBOARD_HUB=y
 CONFIG_VFP=y
+CONFIG_VIDEO_GC2145=m
+CONFIG_VIDEO_OV5640=m
+CONFIG_VIDEO_ST_MIPID02=m
